src/DataTypes/Serializations/SerializationMap.cpp

```diff
@@ -216,6 +216,38 @@ ReturnType SerializationMap::deserializeTextImpl(IColumn & column, ReadBuffer &
     return ReturnType(true);
 }
 
+void SerializationMap::readMapSafe(DB::IColumn & column, std::function<void()> && read_func)
+{
+    size_t initial_size = column.size();
+    try
+    {
+        read_func();
+    }
+    catch (...)
```

Comment:
By the way we still can see that errors in `system.errors`, but since for skipping errors setting `input_format_allow_errors_num` should be set by user, maybe it's even expected

Reply:
Yes, it's expected